components/ParameterForm.vue

```diff
@@ -118,27 +126,36 @@ const renderAsRadios = (parameter: Parameter) => {
   return parameter.parameterType === ParameterType.Select && optionsAreTerse(parameter);
 };
 
-const submitForm = () => {
-  // Not implemented yet
-};
+const formSubmitting = ref(false);
 
-const largeScreen = ref(true);
-const breakpoint = 992; // CoreUI's "lg" breakpoint
-const setFieldSizes = () => {
-  if (window.innerWidth < breakpoint) {
-    largeScreen.value = false;
-  } else {
-    largeScreen.value = true;
-  }
+const submitForm = async () => {
+  if (!formData.value) {
```

Comment:
when does this happen?

Reply:
When does !formData.value be undefined, you mean? It would only happen if the inputs were nullish, which shouldn't happen, but possibly could with some combination of form autofilling and hydration lifecycle race conditions? It's defensive coding.